srv/SetObjectPose.srv

```diff
@@ -0,0 +1,9 @@
+string name
+geometry_msgs/Point position
+float64 yaw
```

Comment:
Since stage shows angles as degrees in the viewer, we should avoid confusion by adding a comment in the srv file where we define that the yaw has to be in radians.